### DISCUSSION TITLE
How to set height for whole grid ?

### DISCUSSION THREAD

**Discussion by evbargavi** [Comcast/react-data-grid#2574] [Q&A]
I have used minHeight property. But still i cant getting the desired table height that i want to assign.

**Comment by okram2k** [comment]
best I've found is you can set the height in the style component as such:
<DataGrid style={{height: 500}} /> would set it a height of 500 pixels. If there's a better way I'd love to know myself. I've been using window.innerHeight to get the window size to make it fit the whole screen.

**Comment by evbargavi** [comment]
HI,

Thanks for the reply.
It works well.

I need another solution.. 

In my site - i am having many columns fr data grid - in which i have to show only columns fit into the container width.
and i have manually hide horizontal scrollbar.
Is there any possibility to show the hided columns in the container?
